pull_request_template.md

```diff
@@ -1,23 +1,28 @@
-#### Feature / Issue / Improvement / Refactor (pick one)
+# Problem description
+Describe here what problem you are solving. Often this is (or should be) the same description is in the Linear ticket.
+Please don't just link to the Linear ticket here, don't force the reviewer to jump back and forth between GH and Linear.
 
+Example:
+> Support for BlockFi is quite broken. Either the imports fail outright (ca. 25% of the cases) or when an unsupported transaction is encountered it is silently ignored, i.e. there is no indication to the user that certain transactions were not imported.
 
-#### Summary of changes
-- 
+# Background
+You've just put a lot of effort into understanding the underlying code before making a change. Share these leanings with
+the reviewer. Assume they don’t know the code very well. If this section becomes too large consider creating a Notion page and only keep a summary here with a link to the page.
 
-#### Considerations
-<!-- Style: Did you review the PR diff to ensure it conforms to our style guide? -->
-<!-- Migrations: Did you handle any relevant migrations for existing users or data? -->
-<!-- Deployment: Are any pre-deployment or post-deployment actions needed? -->
-<!-- Web: Is UI mobile-optimized? -->
+Example:
+> The CSV import logic for BlockFi is pretty straight-forward:
+>- transfers with a positive amount are credits, with a negative amount debits
+>- trades are transactions of type ["Trade", "Ach Trade"] that share the same datetime
 
-### Tests
-<!-- Tests: Are the changes tested with new or existing tests? Did you manually test? -->
-- [ ] Application changes have been tested thoroughly
-- [ ] Automated tests covering modified code pass
 
-#### Security
+# Solution
+Describe in a few sentences the approach you took to address the problem. If this section becomes too large consider creating a Notion page and only keep a summary here with a link to the page.
 
-- [ ] Security impact of change has been considered
```

Comment:
This was required from Vanta when we had a SOC2 audit last year

Reply:
Thanks for pointing this out, will put them back